telegram/_callbackquery.py

```diff
@@ -238,10 +253,13 @@ async def edit_message_text(
                 chat_id=None,
                 message_id=None,
             )
+        if not isinstance(self.message, Message):
```

Comment:
why not have a method like `get_message()` (akin to get_bot()) which raises an error if the message is Inaccessible?

Reply:
for internal use that's a nice idea, I agree 👍 I wouldn't make it public though, b/c there are three different cases (Message, InaccessibleMessage, None) and depending on the use case people might be expecting different behavior …